repositories/README.md

```diff
@@ -0,0 +1,10 @@
+All repository data is imported via the "images" library, see
+https://github.com/osbuild/images/tree/main/data/repositories
+
+Most are imported via the "vendor" mechanism, but some special
+repositories are kept here, like the `*-no-aux-key.json` that
+is only installed on rhel-8.
```

Comment:
This is not true and needs to be "-no-aux-keys.json" is only installed on rhel-10

Reply:
True, this is a typo. Those are el8 repos installed only on el10.

Reply:
Fixed this now with a better explaination.